Automattic/jetpack
# Fix activation and layout for Protect interstatial page on My Jetpack

This fixes a layout problem where the product card was overlapping with the bundle card, and fixes the activation flow when a user clicks the free plan.

## Proposed changes:

* Added `is_upgradable_by_bundle` call to the Protect product class
* Changed slightly the onClick handler for ProductInterstitial to be able to receive the details from the selected product. Since the Protect page has a bundle and also has the free product we need to check know which product was selected that was selected to get the correct `isBundle` property to determine whether to redirect to the checkout or not.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to '/wp-admin/admin.php?page=my-jetpack#/add-protect'
* Verify that the free option installs and activates the plugin
* Verify that the bundle option correctly redirects to the checkout
* Test out other interstatial pages to make sure everything is still working.
